demo/docbot/ingestion.py

```diff
@@ -8,7 +9,20 @@
 # check out the _bulk python client helper.
 
 # Documents should be ingested from the /data/ dir
+DEMOS_PATH = os.path.abspath(os.path.join(__file__,  "..", "..",".."))
+DOCUMENTATION_INDEX_PATH = "data/documentation-index.json"
+WEBSITE_INDEX_PATH = "data/website-index.json"
+
+def read_files_from_data() -> (list, list):
+  documentation_index = os.path.join(DEMOS_PATH, DOCUMENTATION_INDEX_PATH)
+  website_index = os.path.join(DEMOS_PATH, WEBSITE_INDEX_PATH)
+  with open(documentation_index) as f:
+    documentation_index_data = json.load(f)
+  with open(website_index) as f:
+    website_index_data = json.load(f)
+
+  return documentation_index_data, website_index_data
```

Comment:
If we are going to assume more then these two documents may be used as sources then we should change the return type. We can probably safely concatenation these dictionaries together. Search indexes don't need a rigid schema so we can safely store these all in a single index.